test/integration/auth/node_test.go

```diff
@@ -343,6 +353,80 @@ func TestNodeAuthorizer(t *testing.T) {
 	expectAllowed(t, createNode2MirrorPod(superuserClient))
 	expectAllowed(t, createNode2NormalPodEviction(node2Client))
 	expectAllowed(t, createNode2MirrorPodEviction(node2Client))
+
+	// PVC admission tests
+	superuserExternalClient := externalClientSetForToken(tokenMaster, clientConfig)
```

Comment:
just use the existing superuserClient

Reply:
should be able to use the existing PVC object for these update tests

Reply:
hmm, I would rather not use existing PVC object, because then I will have to convert api.XXX to v1.XXX and then dealing with various bit of orderings in which relationship is established and feature flag gets enabled.

Reply:
> then I will have to convert api.XXX to v1.XXX

I don't understand... I expected:
```
	updatePVCCapacity := func(client clientset.Interface) error {
		client.Core().PersistentVolumeClaims("ns").Patch("mypvc", types.MergePatchType, []byte(`{"status":{...}}`), "patch")
	}
```

Reply:
if you want a separate block for the PVC update tests, that's ok, but the existing pattern of setting up fixed objects, and checking results of various clients acting on those objects is what I would expect here

> dealing with various bit of orderings in which relationship is established

the work is already done to set up the objects with established relationships. use different clients to test the behavior with and without a relationship to the object (node1 and nodeanon have no relationship to the PVC, node2 has a relationship)

Reply:
IMO - the test should not have used internal API object representation to begin with. Having to manually specify PATCH string is a internal detail that should never look into test.

Reply:
That was an example. You can do the same thing with an get-then-update. The point of this test is to test authorization, not version-specific aspects of API objects.

Reply:
How do I create a patch string using `api.PersistentVolumeClaim` object that is acceptable by patcher? (apart from hardcoding the status string ).

Reply:
fixed.